packages/near-api-js/test/account.test.js

```diff
@@ -75,19 +78,21 @@ test('findAccessKey returns the same access key when fetched simultaneously', as
 });
 
 describe('errors', () => {
-    let oldLog;
     let logs;
 
     beforeEach(async () => {
-        oldLog = console.log;
         logs = [];
-        console.log = function () {
-            logs.push(Array.from(arguments).join(' '));
-        };
+        configureLogging({
+            showLogs: () => true,
+            onLog: (args) => {
+                logs.push(Array.from(args).join(' '));
+            },
+            onWarn: () => {}, // Do nothing onWarn
+        });
```

Comment:
I think you need to use spread in `args` here to recreate the same behavior as before

Reply:
@esaminu I have tested this behaviour and was working as the previous one.

Reply:
It looks like it wouldn't work if multiple arguments are used. Does it still work in this case?